types/sortable-options.d.ts

```diff
@@ -0,0 +1,204 @@
+import { Sortable } from './sortable';
+import { SortableEvent, SortableMoveEvent } from './sortable-event';
+
+export interface SortableGroupOptions {
```

Comment:
This is a breaking change to those using Typescript currently. Should be renamed to `GroupOptions`.